packages/next-ui/ActionCard/ActionCard.tsx

```diff
@@ -118,15 +131,37 @@ export function ActionCard(props: ActionCardProps) {
         selected &&
           ((theme) => ({
             border: `2px solid ${theme.palette.secondary.main} !important`,
-            borderTopLeftRadius: theme.shape.borderRadius,
-            borderTopRightRadius: theme.shape.borderRadius,
-            borderBottomLeftRadius: theme.shape.borderRadius,
-            borderBottomRightRadius: theme.shape.borderRadius,
+            boxShadow: `0 0 0 4px ${alpha(
+              theme.palette.secondary.main,
+              theme.palette.action.activatedOpacity,
+            )} !important`,
+            borderTopLeftRadius: responsiveVal(
+              theme.shape.borderRadius * 3,
+              theme.shape.borderRadius * 4,
+            ),
+            borderTopRightRadius: responsiveVal(
+              theme.shape.borderRadius * 3,
+              theme.shape.borderRadius * 4,
+            ),
+            borderBottomLeftRadius: responsiveVal(
+              theme.shape.borderRadius * 3,
+              theme.shape.borderRadius * 4,
+            ),
+            borderBottomRightRadius: responsiveVal(
+              theme.shape.borderRadius * 3,
+              theme.shape.borderRadius * 4,
+            ),
             padding: `${theme.spacings.xxs} ${theme.spacings.xs}`,
           })),
         disabled &&
           ((theme) => ({
-            background: theme.palette.background.default,
+            '& *': {
+              opacity: theme.palette.action.disabledOpacity,
+            },
+            background: alpha(
+              theme.palette.action.disabledBackground,
+              theme.palette.action.disabledOpacity / 10,
+            ),
```

Comment:
This seems overly complex

Reply:
When there is an error submitting the form, the disabled actionCard looks out of place since the whole opacity is changed. This fixes that: https://d.pr/i/1G9Nxs